utils.groovy

```diff
@@ -25,4 +25,35 @@ void addRemoteRef(String branchName) {
   }
 }
 
+/**
+  * Get Version
+  */
+def getVersion() {
+  def rstudioVersion = sh(
+                          script: "docker/jenkins/rstudio-version.sh ${params.RSTUDIO_VERSION_PATCH}",
+                          returnStdout: true
+                        ).trim()
+  echo "RStudio build version: ${rstudioVersion}"
+  currentBuild.displayName = "${rstudioVersion}"
```

Comment:
It's possible that this function could get called in the context of a build that wants to set a different display name. Even though I think it's really nice when a build _does_ want to set the display name, we should probably leave it up to the caller

Reply:
Completely agree! Done.